generic_tracers/generic_COBALT.F90

```diff
@@ -3579,9 +3617,15 @@ subroutine generic_COBALT_update_from_source(tracer_list,Temp,Salt,rho_dzt,dzt,h
        bact(1)%jvirloss_n(i,j,k) = bact(1)%temp_lim(i,j,k)*bact(1)%vir*bact(1)%f_n(i,j,k)**2.0
        bact(1)%jvirloss_p(i,j,k) = bact(1)%jvirloss_n(i,j,k)*bact(1)%q_p_2_n
 
-       !
        ! 3.2.3 Calculate losses to exudation
        !
+       ! Phytoplankton are assumed to lose a constant fraction of carbon they fix to dissolved organic nutrients 
```

Comment:
should this be nitrogen here as the model units are in nitrogen? and also because in line 3624 the comment references nitrogen.

Reply:
I think a change to nitrogen would be fine.  I think I put C because, if I recall correctly, that is what Baines and Pace used, but N is indeed more precise for COBALT.